my-code/src/constants/omdbApi.js

```diff
@@ -0,0 +1,6 @@
+const omdbApi = {
+  API_KEY: "a4e8e9b2",
```

Comment:
careful with adding API keys to the repo.
take a look at [dotenv](https://www.npmjs.com/package/dotenv)